x/auth/client/cli/account.go

```diff
@@ -35,14 +35,18 @@ func GetAccountCmd(storeName string, cdc *wire.Codec, decoder sdk.AccountDecoder
 		Use:   "account [address]",
 		Short: "Query account balance",
 		Args:  cobra.ExactArgs(1),
+
 		RunE: func(cmd *cobra.Command, args []string) error {
 
 			// find the key to look up the account
 			addr := args[0]
-			bz, err := hex.DecodeString(addr)
+			hrp, bz, err := bech32cosmos.DecodeAndConvert(addr)
```

Comment:
what does `hrp` mean?

Reply:
maybe something like `bechPrefix` could be a better variable name (if that's an accurate description)

Reply:
Human Readable Part

Reply:
The HRP terminiology is pretty standard across the dozen or so bech32 implementations. Reluctant to change it

Reply:
sounds good! didn't know

Reply:
I'm a fan of including this abbr. in a comment close to it's first definition for us mere muggles (`// get human readable part`)